cosmic-settings/src/pages/desktop/appearance/mod.rs

```diff
@@ -408,6 +408,8 @@ impl Page {
         ]
         .spacing(theme.space_m())
         .width(Length::Fill)
+        .apply(widget::scrollable)
+        .height(Length::Fill)
```

Comment:
These two additions aren't necessary after the recent context drawer changes, and lead to an infinite scrollbar.